src/main/configurations/Translate/Configuration_VoegZaakdocumentToe_Lk01.xml

```diff
@@ -20,12 +20,14 @@
             <PutInSessionPipe 
                 name="StoreZdsZaakDocumentInhoud">
                 <Param name="ZdsZaakDocumentInhoud" xpathExpression="edcLk01/object[1]" type="DOMDOC"/>
-                <Forward name="success" path="CallGetZgwZaak" />
+                <Forward name="success" path="StoreDocumentData" />
             </PutInSessionPipe>
-
+            <EchoPipe name="StoreDocumentData" elementToMove="inhoud" getInputFromSessionKey="ZdsZaakDocumentInhoud" storeResultInSessionKey="ZdsZaakDocumentInhoudDataRef">
+                <Forward name="success" path="CallGetZgwZaak" />
+            </EchoPipe>
```

Comment:
Can this be merged into the PutInSessionPipe? Put the elementToMove attribute on the PutInSessionPipe? Currently the large document is present in both the ZdsZaakDocumentInhoud and in ZdsZaakDocumentInhoudDataRef.

Reply:
![image](https://github.com/ibissource/zaakbrug/assets/6784732/e4ef6608-e008-4f48-a573-7d480d701a69)

The large document is not present in session key ZdsZaakDocumentInhoudDataRef. If I put the elementToMove in the PutInSessionPipe, the pipe will first remove the document data and then it will apply the XPath expression `edcLk01/object[1]`. If there are multiple matches of pattern `object`, the `ref_inhoud` session key may be filled from all the matches. I propose not to change this code.